File: app/src/test/java/com/tomaszrykala/dorefindmi/things/supplier/digidisplay/MockDigiDisplaySupplier.java
package com.tomaszrykala.dorefindmi.things.supplier.digidisplay;

public class MockDigiDisplaySupplier implements DigiDisplaySupplier {

    @Override public void init() {
        System.out.println("MockDigiDisplaySupplier.init");
    }

    @Override public void display(double n) {
        System.out.println("MockDigiDisplaySupplier.display: " + n);
    }

    @Override public void display(String s) {
        System.out.println("MockDigiDisplaySupplier.display: " + s);
    }

    @Override public void close() throws Exception {
        System.out.println("MockDigiDisplaySupplier.close");
    }
}

File: app/src/test/java/com/tomaszrykala/dorefindmi/things/supplier/digidisplay/MockDigiDisplaySupplier.kt
package com.tomaszrykala.dorefindmi.things.supplier.digidisplay

class MockDigiDisplaySupplier : DigiDisplaySupplier {

    override fun display(n: Double) {
        println("MockDigiDisplaySupplier.display: " + n)
    }

    override fun display(s: String) {
        println("MockDigiDisplaySupplier.display: " + s)
    }

    @Throws(Exception::class)
    override fun close() {
        println("MockDigiDisplaySupplier.close")
    }
}
